IntegrationTests/Fixtures/XCBuild/SystemTargets/Inputs/libsys.h

```diff
@@ -0,0 +1 @@
+const char* GetSystemLibName();
```

Comment:
Can you change the test setup to produce the binary artifacts as part of the test? Otherwise, it’s harder to update the test.